containers/sign/PwdLabel.module.scss

```
@@ -0,0 +1,57 @@
+@import '/styles/color';
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.label {
+  margin-bottom: 10px;
+  font-size: 16px;
+  color: var($black-33);
```

```suggestion
color: $black-33;
```
var 사용안해도 변수로 바로 사용 가능합니다!